### PR TITLE
1010CG Content Feedback

### PR DESCRIPTION
**Pull request by LinnJS**
## Description
During the VSP collaboration there was feedback to fix some consistency issues. Fix feedback from VSP [ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/9849)

### Intro Page
Screen Shot 2020-08-24 at 1 30 56 PM

image

## Acceptance criteria
- [x] Fix VSP feedback
- [x] Fix intro casing
- [x] Remove Sex unknown and make it not-required (stakeholder feedback)
- [x] Need explicit approval from VSA Stakeholders

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Comment by LinnJS** [comment]
This PR is on hold due to the BE work needed to merge https://github.com/department-of-veterans-affairs/vets-json-schema/pull/486 which is required to merge this PR.

**Comment by LinnJS** [comment]
All gender work has been isolated to this PR due to it being blocked by BE work
https://github.com/department-of-veterans-affairs/vets-website/pull/13993

**Review by Mottie — APPROVED** [review]

**Comment by LinnJS** [comment]
This is blocked due to it waiting for explicit approval from VA stakeholders

**Pull request merged**
